plugins/catalog-backend/openapi.yaml

```diff
@@ -57,6 +47,7 @@ components:
       required:
         - url
       description: A link to external information that is related to the entity.
+      additionalProperties: false
```

Comment:
JSON Schema doesn't restrict additional properties by default, but I think this makes sense here as we're expecting types to match the defined interfaces and whatnot when interacting with express handlers. Without this, the json schema library we're using adds a `[x: string]: unkown` key to all objects it parses which causes type issues with interfaces.

This may have issues further down the line with validator middleware, but that's pretty speculative.